<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#alert_type_label,#alert_changeable_message{
background-color:transparent;
}</string>
  </property>
  <widget class="QFrame" name="alert_colored_frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#alert_colored_frame{
	background-color: rgb(255, 80, 92);
    border-radius:10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="alert_white_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#alert_white_frame{
	background-color: rgb(244, 244, 244);
   
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="alert_type_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#alert_type_label{
color: rgb(100, 100, 100);
}
</string>
     </property>
     <property name="text">
      <string>ERROR!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="alert_ok_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#alert_ok_button{
border:none;
	background-color: rgb(255, 80, 92);
border-radius:10px;
color:#fff;
}
#alert_ok_button:pressed{
	background-color:rgb(255, 101, 124);
}
#alert_ok_button:hover{
	background-color: rgba(255, 80, 92, 190);
}</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="alert_changeable_message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>431</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Error Message Here....</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="alert_icon">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/icons/pngwing.com.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
